pkg/kubelet/kubelet_server_journal_darwin.go

```diff
@@ -0,0 +1,8 @@
+// +build darwin
```

Comment:
call the file `patch_....go` to make clear that this comes from a carry patch.

Reply:
Should I rename the already-existing `kubelet_server_journal.go` as well? It stems from an earlier carry patch (https://github.com/openshift/origin/pull/21701)